espnet2/enh/espnet_model.py

```diff
@@ -7,17 +8,19 @@
 
 import ci_sdr
 import torch
-from torch_complex.tensor import ComplexTensor
 from typeguard import check_argument_types
 
 from espnet2.enh.decoder.abs_decoder import AbsDecoder
 from espnet2.enh.encoder.abs_encoder import AbsEncoder
 from espnet2.enh.encoder.conv_encoder import ConvEncoder
+from espnet2.enh.layers.complex_utils import is_complex
 from espnet2.enh.separator.abs_separator import AbsSeparator
 from espnet2.torch_utils.device_funcs import force_gatherable
 from espnet2.train.abs_espnet_model import AbsESPnetModel
 
 
+is_torch_1_3_plus = LooseVersion(torch.__version__) >= LooseVersion("1.3.0")
```

Comment:
Don't worry about this because we already dropped pytorch < 1.3.0. https://github.com/espnet/espnet/releases/tag/v.0.10.0

Reply:
Thanks, I didn't notice this. I will remove these checks then.